nifi-nar-bundles/nifi-standard-bundle/nifi-standard-processors/src/main/java/org/apache/nifi/processors/standard/ExtractGrok.java

```diff
@@ -107,31 +120,70 @@
         .build();
 
     public static final PropertyDescriptor CHARACTER_SET = new PropertyDescriptor.Builder()
-        .name("Character Set")
+        .name(CHARACTER_SET_KEY)
         .description("The Character Set in which the file is encoded")
         .required(true)
         .addValidator(StandardValidators.CHARACTER_SET_VALIDATOR)
         .defaultValue("UTF-8")
         .build();
 
     public static final PropertyDescriptor MAX_BUFFER_SIZE = new PropertyDescriptor.Builder()
-        .name("Maximum Buffer Size")
+        .name(MAXIMUM_BUFFER_SIZE_KEY)
         .description("Specifies the maximum amount of data to buffer (per file) in order to apply the Grok expressions. Files larger than the specified maximum will not be fully evaluated.")
         .required(true)
         .addValidator(StandardValidators.DATA_SIZE_VALIDATOR)
         .addValidator(StandardValidators.createDataSizeBoundsValidator(0, Integer.MAX_VALUE))
         .defaultValue("1 MB")
         .build();
 
-    public static final PropertyDescriptor NAMED_CAPTURES_ONLY = new PropertyDescriptor.Builder()
-        .name("Named captures only")
-        .description("Only store named captures from grok")
+     public static final PropertyDescriptor NAMED_CAPTURES_ONLY = new PropertyDescriptor.Builder()
+        .name(NAMED_CAPTURES_ONLY_KEY)
+        .description("Only store named captures from grokList")
```

Comment:
I'm not sure that I understand what 'grokList' means here. I think it's intended to refer to the name of a variable in the Processor? We need to keep in mind that the user of this Processor will not be looking at the code so we need to explain this in a way that makes sense to the user.

Reply:
my bad. i will do something more expressive.